cognite_toolkit/_cdf_tk/feature_flags.py

```diff
@@ -28,6 +28,10 @@ class Flags(Enum):
         "visible": True,
         "description": "Enables support for the populate command",
     }
+    STRICT_VALIDATION: ClassVar[dict[str, Any]] = {  # type: ignore[misc]
+        "visible": True,
+        "description": "For Workflow/Transformations/Function do not fallback to Toolkit credentials when validation-type != 'dev'",
+    }
```

Comment:
Introducing this behind a feature flag, such that we can be in a releasable state.